src/SQLiteDatabaseIntegrationLoader.php

```diff
@@ -85,10 +85,31 @@ public static function load_plugin() {
 		// We also need to selectively load the necessary classes from the plugin.
 		require_once $plugin_directory . '/php-polyfills.php';
 		require_once $plugin_directory . '/constants.php';
-		require_once $plugin_directory . '/wp-includes/sqlite/class-wp-sqlite-lexer.php';
-		require_once $plugin_directory . '/wp-includes/sqlite/class-wp-sqlite-query-rewriter.php';
-		require_once $plugin_directory . '/wp-includes/sqlite/class-wp-sqlite-translator.php';
-		require_once $plugin_directory . '/wp-includes/sqlite/class-wp-sqlite-token.php';
-		require_once $plugin_directory . '/wp-includes/sqlite/class-wp-sqlite-pdo-user-defined-functions.php';
+
+		$new_driver_supported = class_exists( WP_SQLite_Driver::class );
+		$new_driver_enabled   = $new_driver_supported && defined( 'WP_SQLITE_AST_DRIVER' ) && WP_SQLITE_AST_DRIVER;
+
+		if ( $new_driver_enabled ) {
+			require_once $plugin_directory . '/wp-includes/parser/class-wp-parser-grammar.php';
+			require_once $plugin_directory . '/wp-includes/parser/class-wp-parser.php';
+			require_once $plugin_directory . '/wp-includes/parser/class-wp-parser-node.php';
+			require_once $plugin_directory . '/wp-includes/parser/class-wp-parser-token.php';
+			require_once $plugin_directory . '/wp-includes/mysql/class-wp-mysql-token.php';
+			require_once $plugin_directory . '/wp-includes/mysql/class-wp-mysql-lexer.php';
+			require_once $plugin_directory . '/wp-includes/mysql/class-wp-mysql-parser.php';
+			require_once $plugin_directory . '/wp-includes/sqlite-ast/class-wp-sqlite-connection.php';
+			require_once $plugin_directory . '/wp-includes/sqlite-ast/class-wp-sqlite-configurator.php';
+			require_once $plugin_directory . '/wp-includes/sqlite-ast/class-wp-sqlite-driver.php';
+			require_once $plugin_directory . '/wp-includes/sqlite-ast/class-wp-sqlite-driver-exception.php';
+			require_once $plugin_directory . '/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-builder.php';
+			require_once $plugin_directory . '/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-exception.php';
+			require_once $plugin_directory . '/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-reconstructor.php';
```

Comment:
```suggestion
			require_once $plugin_directory . '/wp-includes/sqlite-ast/class-wp-sqlite-information-schema-reconstructor.php';
require_once $plugin_directory . '/wp-includes/sqlite/class-wp-sqlite-pdo-user-defined-functions.php';
require_once $plugin_directory . '/version.php';
```

Reply:
I needed to load those two files, as it complained about missing functions and missing constants.